FMT.ipynb

```diff
@@ -0,0 +1,908 @@
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "denoised_sequences_2 = dada2.methods.denoise_single(fmt_tutorial_demux_2, trim_left = 13, trunc_len = 150)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Viewing denoising stats\n",
+    "\n",
+    "The `denoise_single` commands return basic statistics about the denoising process, and can be visualized with the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><img onload=\"(function(div, url){\n",
+       "if (typeof require !== 'undefined') {\n",
+       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
+       "} else {\n",
+       "    var baseURL = JSON.parse(\n",
+       "        document.getElementById('jupyter-config-data').innerHTML\n",
+       "    ).baseUrl.slice(0, -1);\n",
+       "}\n",
+       "url = baseURL + url;\n",
+       "fetch(url).then(function(res) {\n",
+       "    if (res.status === 404) {\n",
+       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
+       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
+       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
+       "                        '<br /><br />(Interactive output not available on ' +\n",
+       "                        'static notebook viewer services like nbviewer.)';\n",
+       "    } else if (res.status === 409) {\n",
+       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
+       "                        ' to see the visualization.';\n",
+       "    } else if (res.ok) {\n",
+       "        url = res.url;\n",
+       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
+       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
+       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
+       "                        ' target=\\'_blank\\'>new window</a>'\n",
+       "    } else {\n",
+       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
+       "                        ' errors.';\n",
+       "    }\n",
+       "});\n",
+       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-civ2dffr')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
+      ],
+      "text/plain": [
+       "<visualization: Visualization uuid: cae9ddc3-4822-4f24-b9fd-fe86fdcd7bff>"
+      ]
+     },
+     "execution_count": 47,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "denoise_stats = metadata.visualizers.tabulate(denoised_sequences_1.denoising_stats.view(qiime2.Metadata))\n",
```

Comment:
Two times you use the same variable denoise_stats, please make the variables distinctive.